Client/Nimbus/NimbusFeatureFlagLayer.swift

```diff
@@ -0,0 +1,316 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import MozillaAppServices
+
+class NimbusFeatureFlagLayer {
+
+//    struct HomescreenFeatures {
```

Comment:
Ignore commented out code for now. It is for backup testing.

Reply:
Note for our future selves, this commented code relates to: https://mozilla-hub.atlassian.net/browse/FXIOS-4181